src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -0,0 +1,389 @@
+.signpost {
+    display: inline;
+
+    &:hover {
+        cursor: pointer;
+    }
+
+    button {
+        min-width: $clr_baselineRem_1;
+        margin: 0 $clr_baselineRem_0_3;
+        padding: 0;
+        color: $clr-dark-midtone-gray!important;
```

Comment:
why do we need `!important`? We usually avoid using this because it makes customizability from the user side difficult.